chirp/drivers/th_uv8000.py

```diff
@@ -518,21 +512,20 @@ class THUV8000Radio(chirp_common.CloneModeRadio):
     def get_prompts(cls):
         rp = chirp_common.RadioPrompts()
         rp.info = \
-            ('Click on the "Special Channels" toggle-button of the memory '
-             'editor to see/set the upper and lower frequency-mode values.\n')
+            ('Remember the # T-R key is the VFO/MEM toggle.\n')
 
         rp.pre_download = _(
             "Follow these instructions to download the radio memory:\n"
             "1 - Turn off your radio\n"
             "2 - Connect your interface cable\n"
             "3 - Turn on your radio, volume @ 50%\n"
-            "4 - Radio > Download from radio\n")
+            "4 - CHIRP Menu - Radio - Download from radio\n")
```

Comment:
This doesn't really make sense I think because this is the message shown to the user *after* they've already clicked on the thing described in item 4 above. But, by the time this message is displayed, we won't have opened the serial port or anything, so I think it's fine to just say "4 - Click OK" right?

Reply:
Oops, step 4 should just be "OK".
You can edit those if you like, but I just pushed a new commit for it.